styles/main.css

```diff
@@ -27,7 +28,7 @@ a {
 ------------------------------
 --------------------------- */
 
-#about-page, #illustration-page, #webdesign-page, #internal-page {
+#about-page, #work-page, #internal-page {
     position: relative;
```

Comment:
then that means there's something else wrong in your code. your footer shouldn't float up to the middle by default.